### DISCUSSION TITLE
Can't login on self hosted Umami

### DISCUSSION THREAD

**Discussion by AkihiroEmiya** [umami-software/umami#2961] [Q&A]
Hey there !

I'm working for some hours to put Umami on my server and I encountered a **LOT** of trouble, even for building since cPanel is an absolutely pain for that.
I had to build the app on my PC and then transfer the built-in app on the server

The application work until I try to login :
![image](https://github.com/user-attachments/assets/a62dc8f6-5584-4094-8098-91286c6bbd1c)
![image](https://github.com/user-attachments/assets/bd20ed15-1f4c-47d0-aede-2fb28142cb02)

The only clue I have is :
![image](https://github.com/user-attachments/assets/3daf91b2-303c-4d24-bfa8-855cee66c29a)

And this isn't enough I guess. Can I toggle logs somewhere to see what's the exact problem ?
Database URL shouldn't be the problem since the beginning of the start process on the server work correctly until the "script.js" part

**Comment by Noch22** [comment]
Hello,
Same problem here, have you found a solution ?
thank's
